### DISCUSSION TITLE
App for Safari

### DISCUSSION THREAD

**Discussion by ghost** [darkreader/darkreader#11431] [Q&A] [answered]
Is the Safari app still supported? I checked the App Store and it hasn't been updated in 1 year

**Comment by bershanskiy** [comment, marked as answer]
Yes, the Safari version is supported, but there are a few technical differences. We plan to merge the two projects eventually (but there is no specific timeline at this time). If you experience a problem, please open a ticket. Thank you!

**Reply by exastone** [reply]
The safari version is a disaster, hasn't been updated in years and it straight up isn't even worth it from a donation stand point (you can always donate directly to DarkReader if you want to throw money at the project) because of how badly it messes with safari, it is NOT supported anymore than IE6 was supported last going off.

**Reply by ghost** [reply]
I hope that by the statement, "We plan to merge the two projects," it means that you have not made me unnecessarily buy the app in the App Store by making me spend money for nothing
